package.json

```diff
@@ -23,5 +23,8 @@
   "devDependencies": {
     "dtslint": "github:Microsoft/dtslint#production",
     "types-publisher": "Microsoft/types-publisher#production"
+  },
+  "dependencies": {
+    "openxml": "^1.0.2"
   }
```

Comment:
You shouldn't change this file, it is used by ts files in `script/`

Reply:
Not interested in your silly crap!!!

Reply:
Hard to tell if you're joking around, but please address this feedback. 😃